packages/controllers/src/utils/OptionsUtil.ts

```diff
@@ -30,5 +31,18 @@ export const OptionsUtil = {
     }
 
     return socials
+  },
+  isSocialsEnabled() {
+    return (
+      (Array.isArray(OptionsController.state.features?.socials) &&
+        OptionsController.state.features?.socials.length > 0) ||
+      (Array.isArray(OptionsController.state.remoteFeatures?.socials) &&
+        OptionsController.state.remoteFeatures?.socials.length > 0)
```

Comment:
not for this pr maybe, but we should instead just have a single field in the controller with the "final" value between config and remote config.